app/services/experiments/move_to_project_service.rb

```diff
@@ -83,7 +83,7 @@ def move_tags!(my_module)
 
     # recursively move all activities in child associations to new project
     def move_activities!(subject)
-      Activity.where(subject: subject).update!(project: @project)
```

Comment:
do you understand difference between update and update! ?

Reply:
Yes I do. And there is no .update! method for ActiveRecord:Relation.

Reply:
Activity.where(subject: subject).find_each { |activity| activity.update!(project: @project) }